script/extract-cert-ou

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ $# -lt 1 ]; then
+  echo "usage: $(basename "$0") <certificate-name>"
+  exit 1
+fi
+
+CERT_NAME="$1"
+
+exec security find-certificate -a -c "$CERT_NAME" -p | keytool -printcert | grep 'Owner: ' | ruby -e 'print STDIN.readline.sub("Owner: ", "").split(", ").map { |p| p.split("=") }.to_h["OU"]'
```

Comment:
Maybe clean up that last part with the Ruby script and use `awk` or at least clean Ruby. 😁